src/app/api/team-member/[memberId]/route.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { NextResponse } from "next/server";
+
+const getATeamMember = async (req, res) => {
+  try {
+    const {data} = prisma.teamMember.findUnique({
+      where: {
+        id: req.query.memberId
+      }
+    });
+
+    return new NextResponse.json(data);
+  } catch (error) {
+    return new NextResponse.error(error);
```

Comment:
```suggestion
    return new NextResponse.json({
      error: error
      status: 40x
    }) ;
```

NextResponse.error() doesn't take any parameters + it has been removed from Next [docs](https://nextjs.org/docs/app/api-reference/functions/next-response) :)
![image](https://github.com/mlscvitpune/MLSC-VIT-PUNE-WEBSITE-2023/assets/92905626/437264d0-bf8e-4146-95bd-8e15707747e7)